db/schema.rb

```diff
@@ -10,318 +10,317 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170430221501) do
-
-  create_table "api_keys", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.string "key", collation: "utf8_unicode_ci"
-    t.string "app_name", collation: "utf8_unicode_ci"
-    t.integer "user_id"
-    t.string "github_link", collation: "utf8_unicode_ci"
-    t.boolean "is_trusted"
-    t.index ["user_id"], name: "index_api_keys_on_user_id"
+ActiveRecord::Schema.define(version: 20_170_430_221_501) do
```

Comment:
Taking a look at why the FlagLog issue is happening, but in the meantime this looks really weird.